update.sh

```diff
@@ -126,7 +119,7 @@ cd example/app
 
 meteor update --release "${new_meteor_version}"
 
-npm-check-updates --configFilePath /dev/null --upgrade
+npx npm-check-updates --configFilePath /dev/null --upgrade
```

Comment:
Can the use of `npx` be split out into its own PR? The version bump PRs should be essentially fully automatic.

Reply:
Yes I'll split this up.